src/ui/components/Assist/Assist.tsx

```diff
@@ -0,0 +1,206 @@
+import Events from "ui/components/Events";
+import { shouldShowDevToolsNag } from "ui/components/Header/ViewToggle";
+import Confetti from "ui/components/shared//Confetti";
+import hooks from "ui/hooks";
+import { Nag } from "ui/hooks/users";
+import { useDismissNag } from "ui/hooks/users";
+import { UserInfo } from "ui/hooks/users";
+import { useTestInfo } from "ui/hooks/useTestInfo";
+import { getViewMode } from "ui/reducers/layout";
+import { useAppDispatch, useAppSelector } from "ui/setup/hooks";
+import { ViewMode } from "ui/state/layout";
+import {
+  shouldShowAddComment,
+  shouldShowAddCommentToLine,
+  shouldShowAddCommentToNetworkRequest,
+  shouldShowAddCommentToPrintStatement,
+  shouldShowAddUnicornBadge,
+  shouldShowBreakpointAdd,
+  shouldShowBreakpointEdit,
+  shouldShowConsoleNavigate,
+  shouldShowExploreSources,
+  shouldShowInspectComponent,
+  shouldShowInspectElement,
+  shouldShowJumpToCode,
+  shouldShowJumpToEvent,
+  shouldShowLaunchCommandPalette,
+  shouldShowQuickOpenFile,
+  shouldShowRecordReplay,
+  shouldShowSearchSourceText,
+  shouldShowTour,
+  shouldShowUseFocusMode,
+} from "ui/utils/onboarding";
+
+import styles from "./Assist.module.css";
+
+const useNagDismissal = () => {
+  const dismissNag = useDismissNag();
+  const dispatch = useAppDispatch();
+  const info = useTestInfo();
+
+  const dismissTourNag = () => {
+    const initialPrimaryPanel = "events";
+    dispatch(setSelectedPrimaryPanel(initialPrimaryPanel));
+    dismissNag(Nag.DISMISS_TOUR);
+  };
+
+  return { dismissTourNag };
+};
+
+const Assist: React.FC = () => {
+  const [selectedItem, setSelectedItem] = useState(2);
+  const { nags } = hooks.useGetUserInfo();
+  const viewMode = useAppSelector(getViewMode);
+  const showDevtoolsNag = shouldShowDevToolsNag(nags, viewMode);
+  const showConsoleNavigate = shouldShowConsoleNavigate(nags);
+  const showBreakpointAdd = shouldShowBreakpointAdd(nags);
+  const showBreakpointEdit = shouldShowBreakpointEdit(nags);
+  const showAddComment = shouldShowAddComment(nags);
+  const showAddCommentToLine = shouldShowAddCommentToLine(nags);
+  const showAddCommentToNetworkRequest = shouldShowAddCommentToNetworkRequest(nags);
+  const showAddCommentToPrintStatement = shouldShowAddCommentToPrintStatement(nags);
+  const showJumpToCode = shouldShowJumpToCode(nags);
+  const showAddUnicornBadge = shouldShowAddUnicornBadge(nags);
+  const showRecordReplay = shouldShowRecordReplay(nags);
+  const showExploreSources = shouldShowExploreSources(nags);
+  const showSearchSourceText = shouldShowSearchSourceText(nags);
+  const showQuickOpenFile = shouldShowQuickOpenFile(nags);
+  const showLaunchCommandPalette = shouldShowLaunchCommandPalette(nags);
+  const showJumpToEvent = shouldShowJumpToEvent(nags);
+  const showInspectElement = shouldShowInspectElement(nags);
+  const showInspectComponent = shouldShowInspectComponent(nags);
+  const showUseFocusMode = shouldShowUseFocusMode(nags);
```

Comment:
About half of these values aren't being used, so they should be deleted:
```suggestion
  const showConsoleNavigate = shouldShowConsoleNavigate(nags);
  const showBreakpointEdit = shouldShowBreakpointEdit(nags);
  const showAddComment = shouldShowAddComment(nags);
  const showJumpToCode = shouldShowJumpToCode(nags);
  const showAddUnicornBadge = shouldShowAddUnicornBadge(nags);
  const showRecordReplay = shouldShowRecordReplay(nags);
  const showExploreSources = shouldShowExploreSources(nags);
  const showSearchSourceText = shouldShowSearchSourceText(nags);
  const showLaunchCommandPalette = shouldShowLaunchCommandPalette(nags);
  const showJumpToEvent = shouldShowJumpToEvent(nags);
  const showInspectElement = shouldShowInspectElement(nags);
  const showUseFocusMode = shouldShowUseFocusMode(nags);
```

FWIW dead code like this usually shows up as _gray_ in VS Code to help you spot it.

Reply:
Thanks! I didn't know about that VS Code feature, that's going to come in quite handy.